elide-swagger/src/main/java/com/yahoo/elide/swagger/SwaggerBuilder.java

```diff
@@ -727,8 +729,15 @@ public Swagger build() {
         /* Create a Model for each Elide entity */
         Map<String, Model> models = new HashMap<>();
         for (Type<?> clazz : allClasses) {
-            Preconditions.checkState(clazz instanceof ClassType);
-            models.putAll(converters.readAll(((ClassType) clazz).getCls()));
+            if (clazz instanceof ClassType) {
+                models.putAll(converters.readAll(((ClassType) clazz).getCls()));
+            } else if (clazz instanceof Type) {
+                ModelConverterContextImpl context = new ModelConverterContextImpl(Arrays.asList(converter));
+                context.resolve(clazz);
+                models.putAll(context.getDefinedModels());
+            } else {
+                models.putAll(converters.readAll(clazz, null));
```

Comment:
What is the 2nd argument here?

Reply:
removed it. It was `com.fasterxml.jackson.annotation.JsonView`. Internally it uses `null` value if not passed.